advanced_survey/static/advanced-survey/src/direflow-components/state.tsx

```diff
@@ -20,12 +20,12 @@ export const sortByListOrder = (arr: Question[]) => {
 
 const makeID = (length: number) => {
     let result = '';
-    const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+    const characters = '0123456789';
```

Comment:
We use alphabet ids for detecting new questions. If we use int based ids the question creation process in the backend would break, please update backend side.